java/src/main/java/org/rocksdb/WriteBatchWithIndex.java

```diff
@@ -120,20 +120,23 @@ public WBWIRocksIterator newIterator() {
    * the write batch update finishes. The state may recover after Next() is
    * called.
    *
+   * @param read_opts The read options to use
    * @param columnFamilyHandle The column family to iterate over
    * @param baseIterator The base iterator,
    *   e.g. {@link org.rocksdb.RocksDB#newIterator()}
    * @return An iterator which shows a view comprised of both the database
    * point-in-time from baseIterator and modifications made in this write batch.
    */
   public RocksIterator newIteratorWithBase(
+      final ReadOptions read_opts,
```

Comment:
Hmm, ideally we should deal with backward compatibility here too.

Reply:
Done, thanks